benefits/settings.py

```diff
@@ -204,7 +204,7 @@ def RUNTIME_ENVIRONMENT():
 
 USE_I18N = True
 
-TIME_ZONE = "UTC"
+TIME_ZONE = "America/Los_Angeles"
```

Comment:
I get why this setting is needed in this PR as well, and it makes sense to go here rather than #1988 

But can you please include the comments, both in code and in the commit message, that I included in 598c12f29f3a4ad882ec381d4a7347a13de85093? I think it's important context that isn't captured by the commit in this PR.

Reply:
Cherry-picked your commit as [`2570489`](https://github.com/cal-itp/benefits/pull/1989/commits/2570489e43dc0a039ea3af974c8f89cda3366b68)